src/SFA.DAS.EmployerFinance.Web/Filters/GoogleAnalyticsViewBagFilter.cs

```diff
@@ -0,0 +1,23 @@
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Filters;
+using Microsoft.Extensions.DependencyInjection;
+using SFA.DAS.EmployerFinance.Configuration;
+using StructureMap;
+
+namespace SFA.DAS.EmployerFinance.Web.Filters
+{
+    public class GoogleAnalyticsViewBagFilter : ActionFilterAttribute
```

Comment:
Is this filter needed?

Reply:
Not anymore due to that <AT>inject change. this has now been removed